app/src/main/res/values/strings.xml

```diff
@@ -55,6 +56,11 @@
     <string name="create_profile_error_adding_to_database">Error while adding profile to database</string>
     <string name="create_profile_passwords_not_equal">Please make sure you typed the same password twice.</string>
     <string name="create_profile_all_fields_empty">Please fill in the fields</string>
+    <string name="create_profile_invalid_iban">The IBAN number is invalid, please enter a valid one.</string>
+
+    <string name="name_create_profile">Name*</string>
+    <string name="repeat_password">Repeat password*</string>
+    <string name="password_create_profile">Password*</string>
```

Comment:
Why did you change these string names/Why are these not listed under `<!-- Common sentences -->` or `<!-- Words -->`